release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Fixed the multiple login requests issue [PAPP-23430]
```

Comment:
Make this sound more formal, such as:
`Fixed issue where multiple logins were being requested`